src/crud.py

```
@@ -720,34 +941,168 @@ async def get_evaluation_runs(db: AsyncSession) -> List[EvaluationRun]:
+        for prompt_config in run.prompt_configurations:
+            # Add family_id and version fields that the schema expects
+            if prompt_config.prompt_version:
+                prompt_config.family_id = prompt_config.prompt_version.family_id
+                prompt_config.version = prompt_config.prompt_version.version
+            else:
+                prompt_config.family_id = None
+                prompt_config.version = None
+    
+    return run
 
 async def get_evaluation_comparison(db: AsyncSession, run_id: int) -> Optional[Dict[str, Any]]:
```

Pylint (3.3.7)

[refactor] Too many local variables (32/15)

(R0914)